index.html

```diff
@@ -263,5 +263,25 @@ <h1 class="first-title2">About Myself</h1>
         </div>
     </div>
 </section>
+<footer class="footer" id="contact">
+    <div class="container-3">
+        <div class="parent">
+            <img src="./Images/contactformshapes-mobile.png" class="over-img" alt="contactformimage"/>
+        </div>  
+    <h1 class="first-title3">Contact me</h1>
```

Comment:
- [ Optional] You should use only one `<h1>` heading tag in an HTML document for semantic reasons. And that one should go to the most important heading the HTML page. This helps for SEO and accessibility for the page.